### DISCUSSION TITLE
evil/:catch: Cannot open load file: No such file or directory, calc-loaddefs.el

### DISCUSSION THREAD

**Discussion by cliffordxp** [emacs-evil/evil-collection#677] [Q&A]
Hi, 
got this error message: 
Error (use-package): evil/:catch: Cannot open load file: No such file or directory, calc-loaddefs.el

Can you check please ?

**Comment by condy0919** [comment]
```elisp
(require 'calc-loaddefs)
```

in scratch buffer. If error still happens, it means your Emacs is broken. Reinstall please, the file `calc-loaddefs.el` is missing in Emacs.
